src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ClaimantTellSomethingElseService.java

```diff
@@ -121,28 +171,203 @@ public String buildApplicationCompleteResponse(CaseData caseData) {
+        String email = userIdamService.getUserDetails(userToken).getEmail();
+        CaseData caseData = caseDetails.getCaseData();
+        String applicationType = caseData.getClaimantTseSelectApplication();
+
+        if (CLAIMANT_TSE_ORDER_A_WITNESS_TO_ATTEND.equals(applicationType)) {
+            emailService.sendEmail(
+                    tseClaimantRepAcknowledgeTypeCTemplateId,
+                    email,
+                    prepareEmailContentTypeC(caseDetails));
+            return;
+        }
+
+        if (NO.equals(caseData.getClaimantTseRule92())) {
+            emailService.sendEmail(
+                    tseClaimantRepAcknowledgeNoTemplateId,
+                    email,
+                    prepareEmailContent(caseDetails));
+            return;
+        }
+
+        emailService.sendEmail(
+                GROUP_B_TYPES.contains(applicationType)
+                        ? tseClaimantRepAcknowledgeTypeBTemplateId
+                        : tseClaimantRepAcknowledgeTypeATemplateId,
+                email,
+                prepareEmailContent(caseDetails));
+    }
+
+    private Map<String, String> prepareEmailContentTypeC(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+        return Map.of(
+                CASE_NUMBER, caseData.getEthosCaseReference(),
+                CLAIMANT, caseData.getClaimant(),
+                RESPONDENT_NAMES, getRespondentNames(caseData),
+                EXUI_CASE_DETAILS_LINK, emailService.getExuiCaseLink(caseDetails.getCaseId())
+        );
+    }
+
+    private Map<String, String> prepareEmailContent(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+
+        Map<String, String> content = new ConcurrentHashMap<>();
+        content.put(CASE_NUMBER, caseData.getEthosCaseReference());
+        content.put(CLAIMANT, caseData.getClaimant());
+        content.put(RESPONDENT_NAMES, getRespondentNames(caseData));
+        content.put(HEARING_DATE, getNearestHearingToReferral(caseData, NOT_SET));
+        content.put(SHORT_TEXT, caseData.getClaimantTseSelectApplication());
+        content.put(EXUI_CASE_DETAILS_LINK, emailService.getExuiCaseLink(caseDetails.getCaseId()));
+
+        return content;
+    }
+
+    public void sendRespondentsEmail(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+        String applicationType = caseData.getClaimantTseSelectApplication();
+
+        if (CLAIMANT_TSE_ORDER_A_WITNESS_TO_ATTEND.equals(applicationType)
+            || NO.equals(caseData.getClaimantTseRule92())
+            || YES.equals(caseData.getClaimantTseRespNotAvailable())) {
+            return;
+        }
+
+        List<String> respondentEmailAddressList = getRespondentEmailAddressList(caseData);
+        boolean isWelsh = featureToggleService.isWelshEnabled()
+                && WELSH_LANGUAGE.equals(caseData.getClaimantHearingPreference().getContactLanguage());
+
```

Comment:
Maybe add just in case there are no valid email addresses to send emails to 
```suggestion
if (respondentEmailAddressList.isEmpty()) {
    return;
}
```